storage/innobase/handler/ha_innodb.cc

```diff
@@ -3880,7 +3880,8 @@ innobase_init(
 	if (!innodb_empty_free_list_algorithm_backoff_allowed(
```

Comment:
There are some code smells in innodb_empty_free_list_algorithm_backoff_allowed: 1) the name hardcodes that it is  "backoff", but the 1st arg passes the actual algorithm to verify; 2) the calculation innobase_buffer_pool_size / srv_page_size / innobase_buffer_pool_instance is the only correct way to pass the 2nd arg.

Thus:
- Please rename to innodb_empty_free_list_algorithm_allowed
- Please inline and remove the 2nd arg